.github/workflows/spellCheck.yaml

```diff
@@ -26,29 +67,48 @@ jobs:
     steps:
     - name: check-spelling
       id: spelling
-      uses: check-spelling/check-spelling@v0.0.20
+      uses: check-spelling/check-spelling@v0.0.21
       with:
         suppress_push_for_open_pull_request: 1
         checkout: true
+        check_file_names: 1
+        spell_check_this: check-spelling/spell-check-this@prerelease
         post_comment: 0
-        dictionary_source_prefixes: '{"cspell": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20220427/dictionaries/","cspell1": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20220814/dictionaries/"}'
+        use_magic_file: 1
+        extra_dictionary_limit: 10
         extra_dictionaries:
-          cspell:powershell/powershell.txt
-          cspell:typescript/typescript.txt
-          cspell:lorem-ipsum/dictionary.txt
-          cspell1:software-terms/src/software-terms.txt
+          cspell:software-terms/src/software-terms.txt
+          cspell:filetypes/filetypes.txt
```

Comment:
When I refresh, I start from scratch and select things roughly in the order that the action suggests.

For reasons I'm not entirely certain, powershell / typescript / lorem were not highly ranked.

I suspect the reason lorem isn't highly ranked is that the lorem pattern already masked all the interesting lorem items. It's possible that the pattern could be removed and the dictionary used instead, but, it doesn't really matter.